src/excalidraw-app/index.tsx

```diff
@@ -96,20 +93,20 @@ const shouldForceLoadScene = (
       }: { room: string; timestamp: number } = JSON.parse(collabForceLoadFlag);
       // if loading same room as the one previously unloaded within 15sec
       //  force reload without prompting
-      if (previousRoom === roomId && Date.now() - timestamp < 15000) {
+      if (
+        previousRoom === roomLinkData.roomId &&
+        Date.now() - timestamp < 15000
```

Comment:
Wow, what is this thing. Looks like this was added during the big refactoring. We need to remove this. https://github.com/excalidraw/excalidraw/pull/2313/files#r569601886

Reply:
I've added this a long time ago. It was useful back then, but no longer indeed.